cylc/flow/scripts/show.py

```diff
@@ -346,9 +347,15 @@ async def prereqs_and_outputs_query(
                     attrs.append("queued")
                 if t_proxy['isRunahead']:
                     attrs.append("runahead")
+                if (
+                    t_proxy['runtime']['runMode']
+                    and t_proxy['runtime']['runMode'] == 'Skip'
+                ):
+                    attrs.append("skip")
```

Comment:
* Not especially obvious what `(skip)` means. Perhaps `run-mode=skip`.
* Could also loosen this to cover other run modes, i.e. `and not t_proxy[runtime]['runMode'] == 'live'`...
* Plz, check this value updates with broadcasts.

Reply:
> * Not especially obvious what `(skip)` means. Perhaps `run-mode=skip`.

Yes. Will do.

> * Could also loosen this to cover other run modes, i.e. `and not t_proxy[runtime]['runMode'] == 'live'`...

This would require change to the data stored by sim and dummy modes. Because they are only settable for the workflow the logic to store the info about the tasks isn't there. Will have a look.

> * Plz, check this value updates with broadcasts.

Putting a test up